### PR TITLE
Adding raft-tick-interval to docs

### PR DESCRIPTION
**Pull request by jseldess**
Adding flag definition to `start-a-node.md` (`cockroach start` command), and calling out flag in the replication recommendations section of `configure-replication-zones.md` (`cockroach zone` command).

cc: @bdarnell

---
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/docs/286)

### CODE REVIEW

File: configure-replication-zones.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ When running a cluster with more than one node, each replica will be on a differ
 
 - Configurations with odd numbers of replicas are more robust than those with even numbers. Clusters of three and four nodes can each tolerate one node failure and still reach a quorum (2/3 and 3/4 respectively), so the fourth replica doesn't add any extra fault-tolerance. To survive two simultaneous failures, you must have five replicas.
 
-- When replicating across datacenters, you should use to datacenters on a single continent to ensure better peformance. Cross-continent and other high-latency scenarios will be better supported in the future. 
+- When replicating across datacenters, you should use datacenters on a single continent to ensure peformance (cross-continent scenarios will be better supported in the future). Also, you should adjust the [`raft-tick-interval`](start-a-node.html#flags) flag on each node to prevent excessive replica leader elections, during which operations on affected replicas stop.
```
Comment (bdarnell) [inline configure-replication-zones.md]: We don't want to encourage this too strongly - most people won't need it. I would say something like "If the average network round-trip latency between your datacenters is greater than 100ms, you should adjust the `raft-tick-interval` flag on each node."

File: start-a-node.md [hunk 2]
```diff
@@ -37,6 +37,7 @@ Flag | Description
...
 `--key` | The path to the [node key](create-security-certificates.html) protecting the node certificate. This flag is required to start a secure node. 
 `--port`<br>`-p` | The port to listen on for internal and client communication. <br><br>**Default:** 26257
+`--raft-tick-interval` | CockroachDB uses the [Raft consensus algorithm](https://raft.github.io/) to replicate data consistently according to your [replication zone configuration](configure-replication-zones.html). For each replica group, an elected leader heartbeats its followers and keeps their logs replicated. When followers fail to receive heartbeats, a new leader is elected. <br><br>This flag sets the interval at which the replica leader heartbeats followers. For high-latency environments, you can adjust this interval to prevent leader elections, during which the operations on affected replicas stop.<br><br>**Default:** 100ms
```
Comment (bdarnell) [inline start-a-node.md]: This should say specifically that it should be set to a value greater than the average network latency between your nodes. Also if this flag is set anywhere, it should be set to the same value on all nodes.

**Comment by jseldess** [comment]
Review status: 0 of 2 files reviewed at latest revision, 2 unresolved discussions.

---

*[configure-replication-zones.md, line 53 \[r2\]](https://reviewable.io:443/reviews/cockroachdb/docs/286#-KHVnot_bF8ZQjDwRKYS-r2-53:-KHWo48Cwy3qIe48Q2Uh:-499146009) ([raw file](https://github.com/cockroachdb/docs/blob/ca32eb2629c28507a4302d44c91d29a60dbeab0d/configure-replication-zones.md#L53)):*
<details><summary><i>Previously, bdarnell (Ben Darnell) wrote…</i></summary>
> We don't want to encourage this too strongly - most people won't need it. I would say something like "If the average network round-trip latency between your datacenters is greater than 100ms, you should adjust the `raft-tick-interval` flag on each node." </details>
Done.

---

*[start-a-node.md, line 40 \[r2\]](https://reviewable.io:443/reviews/cockroachdb/docs/286#-KHVnotZNYYRRDcQs8d5-r2-40:-KHWo7_U85g_Yef52LxC:-499146009) ([raw file](https://github.com/cockroachdb/docs/blob/ca32eb2629c28507a4302d44c91d29a60dbeab0d/start-a-node.md#L40)):*
<details><summary><i>Previously, bdarnell (Ben Darnell) wrote…</i></summary>
> This should say specifically that it should be set to a value greater than the average network latency between your nodes. Also if this flag is set anywhere, it should be set to the same value on all nodes.</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/286)*

**Comment by d4l3k** [comment]
:lgtm:

<details><summary><i>Previously, jseldess wrote…</i></summary>
> #### Adding raft\-tick\-interval to docs
> 
> Adding flag definition to `start-a-node.md` (`cockroach start` command), and calling out flag in the replication recommendations section of `configure-replication-zones.md` (`cockroach zone` command).
> 
> 
> 
> cc: @bdarnell
> 
> 
> </details>


---

Review status: 0 of 2 files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/286#-:-KHWqA0zmso3yEMIG9aW:1417088721)*

**Pull request merged**